portability-api/src/main/java/org/datatransferproject/api/action/transfer/CreateTransferJobAction.java

```diff
@@ -137,7 +137,9 @@ public TransferJob handle(CreateTransferJob request) {
             }
 
             return new TransferJob(encodedJobId, job.exportService(), job.importService(), job.transferDataType(),
-                    exportConfiguration.getUrl(), importConfiguration.getUrl());
+                    exportConfiguration.getUrl(), importConfiguration.getUrl(),
```

Comment:
Just to clarify, what's url vs tokenUrl?

Reply:
Updated url to authUrl